cellxgene_schema_cli/cellxgene_schema/ontology.py

```diff
@@ -1,86 +1,201 @@
 """Methods for working with ontologies and the OLS."""
-from urllib.parse import quote_plus
+import os
+import gzip
+import json
+
+
+class geneChecker:
+    """Handles checking gene ids, retrieves symbols"""
+
+    GENE_FILES = {
+        "Homo sapiens": os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            "ontology_files/genes_homo_sapiens.csv.gz",
+        ),
+        "Mus musculus": os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
```

Comment:
Create a var (PACKAGE_ROOT) for the root directory in the env file and import it and build on it here instead of using os.path.dirname(os.path.realpath(__file__)),

Reply:
fixed in 4ac358decef7509c5944b70e93c9bd1e22eb45c1